placebo/serializer.py

```diff
@@ -53,8 +66,13 @@ def serialize(obj):
         result['second'] = obj.second
         result['microsecond'] = obj.microsecond
         return result
-    if isinstance(obj, StreamingBody):
-        result['body'] = obj.read()
+    # Convert objects to dictionary representation based on type
+    if isinstance(obj, botocore.response.StreamingBody):
+        result['payload'] = obj.read() 
```

Comment:
Why are we switching the key from `body` to `payload`?  This changes the contract of what's stored in a cassette, and maybe we should hold off on that until we do a major version upgrade incase other things (third party tools) depend on it... unless there's a good reason, of course. :-)